tests/lean/macroStack.lean.expected.out

```diff
@@ -6,7 +6,10 @@ while expanding
   x > 0 
 while expanding
   if h:(x > 0) then 1 else 0 
-macroStack.lean:11:9: error: elaboration function for 'Lean.Parser.Term.liftMethod' has not been implemented
+macroStack.lean:11:9: error: invalid use of `(<- ...)`, must be nested inside a 'do' expression
+macroStack.lean:11:6: error: don't know how to synthesize placeholder
+context:
+⊢ Sort u_1
```

Comment:
@leodemoura Can we do something against this extraneous error message?

Reply:
I cannot see it in my branch. I will take a look after I merge this PR